crates/biome_css_formatter/src/css/properties/generic_property.rs

```diff
@@ -12,6 +12,45 @@ impl FormatNodeRule<CssGenericProperty> for FormatCssGenericProperty {
             value,
         } = node.as_fields();
 
+        let is_dash_identity = name.clone().unwrap().as_css_dashed_identifier().is_some();
```

Comment:
Don't use `unwrap()`; you rarely need to use this method because it panics at runtime. In your case you can use `CssDashedIdentifer::can_cast(name.syntax().kind())`;